docs/src/main/asciidoc/container-image.adoc

```diff
@@ -19,6 +19,7 @@ Quarkus provides extensions for building (and pushing) container images. Current
 
 The extension `quarkus-container-image-jib` is powered by https://github.com/GoogleContainerTools/jib[Jib] for performing container builds.
 The major benefit of using Jib with quarkus, is that all dependencies (everything found under target/lib) are cached in a different layer than the actual application making rebuilds extra fast and extra small (when it comes to pushing).
+Another important benefit of using this extension is that it can be used to build a container image without having to have any dedicated client side tooling (like Docker).
```

Comment:
not only tooling, also a running daemon.

Reply:
very good point, I added it!